mcux/mcux-sdk/drivers/cache/cache64/fsl_cache.h

```diff
@@ -79,12 +79,13 @@ uint32_t CACHE64_GetInstance(CACHE64_POLSEL_Type *base);
 #endif
 
 /*!
- * brief Returns an instance number given physical memory address.
+ * brief Returns an instance numbers given physical memory address.
  *
  * param address The physical memory address.
+ * param pointer to store the index of the physical memory index
  * @return CACHE64_CTRL instance number starting from 0.
  */
-uint32_t CACHE64_GetInstanceByAddr(uint32_t address);
+uint32_t CACHE64_GetInstanceByAddr(uint32_t address, uint8_t *bufIndex);
```

Comment:
why is the api changing?

Reply:
This is an internal function and not an API change. This function was not functioning as intended. This function was returning a value which was used as an index for 2 arrays.

Reply:
if it is an internal function, then why is it defined in the header file? 

Is the SDK team okay with this change?

Reply:
Working with the SDK team on this change.